common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -795,6 +795,9 @@ public static enum ConfVars {
         "String used as a file extension for output files. \n" +
         "If not set, defaults to the codec extension for text files (e.g. \".gz\"), or no extension otherwise."),
 
+    HIVE_TEZ_FOR_LOAD_NON_NATIVE_TABLE("hive.tez.for.load.non.native.table", false,
```

Comment:
wouldn't it be better something like 'HIVE_LOAD_DATA_USE_NATIVE_API("hive.load.data.use.native.api")' - use native api instead of tez job ... ?